sepal_ui/sepalwidgets/inputs.py

```diff
@@ -102,7 +106,7 @@ def check_date(self, change):
         self.date_text.error_messages = None
 
         # exit immediately if nothing is set
-        if change["new"] is None:
+        if not change["new"]:
```

Comment:
I try my best not to use `not` like this anymore as `""`, `False` and `None` will all be treated the same way. Is there any reason why you removed the default value of the textField ?

Reply:
ok, so for this very first point, I use to bind the `DatePicker` `v_model` with a `traitlets.Unicode` in the model, Unicode because it is what the widget returns, if we use `None` as the default "`v_model`" it will complain the trait link (we can create the `Unicode(allow_none=True)` tho), so, with this change I —this is just my personal preference— just don't like the `change["new"] == ""`, that's why I've changed it.

And for the second point, you  mean the `v_model` from the `TextField`? I've removed it because it will be linked by chain events by the `link`, so I don't think it is fully neccessary.
```python
        link((self.date_picker, "v_model"), (self.date_text, "v_model"))
```

what do you think?.

Reply:
I agree with making full use of the traitlets and the type checking so big no for the `allow_none=True` I dived into the PEP 8 recommendation and apprently: 

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false.

So as the Unicode cannot be anything else than a string that's legit. 

Could you add a [`versionchanged`](https://pypi.org/project/Deprecated/) to the class to notify this change (I'm positive I'm checking `is None` in some modules) ?